editor.planx.uk/src/components/Feedback/index.tsx

```diff
@@ -355,10 +360,11 @@ const Feedback: React.FC = () => {
 
   return (
     <Box>
-      <FeedbackPhaseBannerView />
-      <Drawer 
+      <FeedbackPhaseBannerView handleFeedbackViewClick={handleFeedbackViewClick}/>
+      <Drawer
         aria-label="Feedback triage and submission form"
-        open={isDrawerOpen} 
+        open={isDrawerOpen}
+        onClose={closeDrawer}
```

Comment:
Passing a handler here for `onClose` ensures that the modal works as expected when there's a click away or the `esc` button is hit.